src/index.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable object-shorthand */
```

Comment:
- [ ] You have done great implementing the suggestions from the previous CR and also meeting the project requirements :+1: :+1: , But there is a change you need to make before moving to the next milestone. :+1: 
- [ ] As required by Microverse, you are not meant to use any linters diable rule in your codebase. The linters are there to make sure your code meets the best standard practices. I'll suggest you remove this linter diable rule and fix the issue. :+1: :+1: :+1: